### DISCUSSION TITLE
Add /echest to /cmi enderchest alias

### DISCUSSION THREAD

**Discussion by Insprill** [Zrips/CMI#2789] [Feature Requests]
**Description of feature/improvement request:**
Add /echest to /cmi enderchest alias
---
### Detailed explanation of what it should do, when and in what situation: ###
Other players and I go to type /echest all the time and it's not a thing (i made it a thing with custom alias but it should be normal)
### Basic explanation as to why this feature is needed and/or useful to have: ###
I think it's a pretty common thing /echest so it would be nice to have

**Comment by mrfloris** [comment]
The "e" from echest is purely from EssentialsX, a customAlias.yml entry can solve this like you say. As server owner you only have to add it once, personally I am not so sure why it has to become a default plugin. Not everybody is converting from Essentials to CMI of course. 

While I understand it would be convenient to support all alias-commands from every other plugin CMI can convert from, I personally think it's not needed. I vote 'not needed in my humble opinion' on this one, but am not against it if it would end up being added.

**Comment by Insprill** [comment]
i didn't even know it was from EssX lol. i just feel like echest is really common

**Comment by Va1a** [comment]
Yeah just use /cmi aliaseditor and add an entry for echest or whatever else you want. the aliaseditor is a very powerful tool.
